::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2018 Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Wed, 30 Aug 2017 22:15:07 +0200
Subject: [PATCH] baremetal, virtual: Extend arch list for UCA repo

JIRA: https://jira.opnfv.org/browse/ARMBAND-304

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 mcp/reclass/classes/cluster/mcp-pike-common-ha/openstack_init.yml   | 2 +-
 mcp/reclass/classes/cluster/mcp-pike-common-noha/openstack_init.yml | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/mcp/reclass/classes/cluster/mcp-pike-common-ha/openstack_init.yml b/mcp/reclass/classes/cluster/mcp-pike-common-ha/openstack_init.yml
index d2d611c..0a3fc6f 100644
--- a/mcp/reclass/classes/cluster/mcp-pike-common-ha/openstack_init.yml
+++ b/mcp/reclass/classes/cluster/mcp-pike-common-ha/openstack_init.yml
@@ -190,7 +190,7 @@ parameters:
       repo:
         uca:
           source: "deb http://ubuntu-cloud.archive.canonical.com/ubuntu xenial-updates/${_param:openstack_version} main"
-          architectures: amd64
+          architectures: amd64,arm64
           key_id: EC4926EA
           key_server: keyserver.ubuntu.com
       kernel:
diff --git a/mcp/reclass/classes/cluster/mcp-pike-common-noha/openstack_init.yml b/mcp/reclass/classes/cluster/mcp-pike-common-noha/openstack_init.yml
index 46d7af4..076f38a 100644
--- a/mcp/reclass/classes/cluster/mcp-pike-common-noha/openstack_init.yml
+++ b/mcp/reclass/classes/cluster/mcp-pike-common-noha/openstack_init.yml
@@ -96,7 +96,7 @@ parameters:
       repo:
         uca:
           source: "deb http://ubuntu-cloud.archive.canonical.com/ubuntu xenial-updates/${_param:openstack_version} main"
-          architectures: amd64
+          architectures: amd64,arm64
           key_id: EC4926EA
           key_server: keyserver.ubuntu.com
       kernel:
